graphics/mupdf/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                mupdf
-version             1.17.0
+version             1.19.1
 revision            1
```

Comment:
the "revision" field should be reset to zero when updating the version

Reply:
Fixed.